crates/curp/src/server/raw_curp/mod.rs

```diff
@@ -1659,12 +1659,20 @@ impl<C: Command, RC: RoleChange> RawCurp<C, RC> {
             return false;
         }
 
-        let replicated_cnt = self
-            .lst
-            .iter()
-            .filter(|f| !f.is_learner && f.match_index >= i)
-            .count();
-        replicated_cnt + 1 >= quorum(self.ctx.cluster_info.voters_len())
+        let voter_len = self.ctx.cluster_info.voters_len();
```

Comment:
It seems the `lst` and `ctx.cluster_info` both stored the voter infomations? I suggest to use `lst` on leader for consistency.

Reply:
The single node mode is removed in that we don't fix this issue in 0.7.0.